protocol/http_scanner.go

```diff
@@ -36,9 +36,18 @@ func (hps * HTTPProtocolScanner) ScanProtocol(hiddenService string, proxyAddress
 			return
 		}
 
+		// Reading all http headers
+		log.Printf("HTTP response headers: %s\n", report.ServerVersion)
```

Comment:
We shouldn't print out server version here. This line could probably be removed.